apps/server/src/modules/files-storage/controller/api-test/files-storage-download-upload.api.spec.ts

```diff
@@ -371,18 +408,25 @@ describe('files-storage controller (API)', () => {
 	describe('file-security.download()', () => {
 		describe('with bad request data', () => {
 			it('should return status 404 for wrong token', async () => {
-				await api.postUploadFile(`/file/upload/${validId}/schools/${validId}`);
 				const response = await api.getDownloadFile('/file-security/download/test-token');
 
 				expect(response.status).toEqual(404);
 			});
 		});
 
 		describe(`with valid request data`, () => {
-			it('should return status 200 for successful download', async () => {
+			const setup = async () => {
+				const expectedResponse = createFileResponse('bytes 0-3/4');
+				s3ClientAdapter.get.mockResolvedValueOnce(expectedResponse);
+
 				const { result } = await api.postUploadFile(`/file/upload/${validId}/schools/${validId}`);
+				const newRecord: FileRecord = await em.findOneOrFail(FileRecord, result.id);
```

Comment:
Why did you specify the type here? We do it hardly anywhere when defining variables. Personally I don't like it.

Reply:
It was needed by Eslint or TS. Not sure